<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <link href='https://fonts.googleapis.com/css?family=Chivo:900' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen" />
    <link rel="stylesheet" type="text/css" href="stylesheets/pygment_trac.css" media="screen" />
    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print" />
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <style>
    .default-spacer{
            margin-right:10px;
    }
    .large-spacer{
            margin-right:100px;
    }
    </style>
    <title>VizBin - Fast and easy human-augmented binning</title>
  </head>

  <body>
    <div id="container">
      <div class="inner">

        <header>
          <h1>VizBin</h1>
          <h2>The fast and easy-to-use application for human-augmented binning</h2>
        </header>

        <section id="downloads" class="clearfix">
        <!---
          <a href="https://github.com/claczny/VizBin/zipball/master" id="download-zip" class="button"><span>Download .zip</span></a>
          <a href="https://github.com/claczny/VizBin/tarball/master" id="download-tar-gz" class="button"><span>Download .tar.gz</span></a>
          --->
          <a href="https://github.com/claczny/VizBin/blob/master/VizBin-dist.jar?raw=true" id="download-jar" class="button"><span>Download app</span></a>
          <a href="https://github.com/claczny/VizBin" id="view-on-github" class="button"><span>View on GitHub</span></a>
        </section>

        <hr>

        <section id="main_content">
          <h3>
<a name="welcome-to-the-vizbin-page" class="anchor" href="#welcome-to-the-vizbin-page"><span class="octicon octicon-link"></span></a>Welcome to the VizBin page.</h3>

<p>You can find news and updates about our application for human-augmented binning here.</p>

<p>Stay tuned for updates!</p>
          
        <h3>
<a name="the-application" class="anchor" href="#the-application"><span class="octicon octicon-link"></span></a>The application.</h3>
<p>Great news! The Software article manuscript <a href="http://www.microbiomejournal.com/content/3/1/1/abstract">VizBin - an application for reference-independent visualization and human-augmented binning of metagenomic data</a> has been accepted for publication in Microbiome!!! As of now, it is the author copy and the final version will follow soon. The "Download app" button always points to the most recent version of VizBin. The version of VizBin that is part of the manuscript can be downloaded <a href="https://github.com/claczny/VizBin/raw/v0.9/VizBin-dist.jar">here</a>.</p>

<p>Below, you can see the screenshot of the visualization window. Each point represents a genomic fragment (here of length >= 1,000nt). The colors are added for visualization purposes only and each color represents an invdividual taxon of origin. The points contained in the red polygon are selected.</p>
<img src="https://github.com/claczny/VizBin/wiki/Tutorials/pictures/13.png" alt="img:polygonal selection">

        <h3>
<a name="usage" class="anchor" href="#usage"><span class="octicon octicon-link"></span></a>Usage.</h3>
<p>VizBin is designed with the user in mind. Hence, we kept the default user-interface simple. No fiddling with parameters. All that is needed is a fasta file containing the sequences of interest. A step-by-step guide on using VizBin - including a description of loading the data, selecting points, and exporting the sequences represented by the selected points - is provided on the <a href="https://github.com/claczny/VizBin/wiki/Tutorial">tutorial page</a> of VizBin's github wiki.</p>
<p>We suggest to visualize fragments that are at least 1,000nt long. Visualizing shorter fragments is also possible but may lead to blurring of distinct clusters, hence the suggested threshold. Filtering sequences based on their length can be done directly from within VizBin, as we explain in the <a href="">tutorial including filtering</a>. Thus, no prior separate filtering step is required.</p>
<h4>Example datasets</h4>
<ul>
    <li><a href="http://claczny.github.io/VizBin/data/37A.zip">37A</a></li>
    <li><a href="http://claczny.github.io/VizBin/data/37B.zip">37B</a></li>
    <li><a href="http://claczny.github.io/VizBin/data/SRS013705.zip">SRS013705</a></li>
    <li><a href="http://claczny.github.io/VizBin/data/AMFJ01.zip">AMFJ01</a></li>
</ul>

          <h3>
<a name="the-concept" class="anchor" href="#the-concept"><span class="octicon octicon-link"></span></a>The concept.</h3>
<p>We explain the concept underlying the intuitive visualization  in detail in the publication <a href="http://www.nature.com/srep/2014/140331/srep04516/full/srep04516.html" ><span>Alignment-free Visualization of Metagenomic Data by Nonlinear Dimension Reduction</span></a>. We also include experiments and results on datasets of varying complexity and size. </p>

<h4>Gist</h4>
<p>The gist of the concept is that VizBin uses the state-of-the-art nonlinear dimension reduction algorithm <a href="http://homepage.tudelft.nl/19j49/t-SNE.html"><span>BH-SNE</span></a> and appropriate data transformation to visualize (assembled) metagenomic data-inherent clusters. 
<h4>Speed</h4>
<p>Due to the efficiency of BH-SNE, VizBin can be used on regular desktop computers and returns results very fast (for 30,000 datapoints, it takes less than 4 minutes on a MacBook Pro from 2011). To achieve further speedups, we have integrated parallelization through the <a href="http://openmp.org/wp/">OpenMP API</a> thus allowing to use multiple cores which are included in most modern computers.</p>

        </section>

        <hr>

        <section id="acknowledgments">
        <h2>Acknowledgments</h2>
        <p>Many people have collaborated on making this application become a reality: Cedric C. Laczny, Tomasz Sternal, Valentin Plugaru, Piotr Gawron, Arash Atashpendar, Hera Houry Margossian, Sergio Coronado, Laurens van der Maaten, Nikos Vlassis, and Paul Wilmes.</p>
        <p>The present work was supported by an ATTRACT programme grant (A09/03) and a European Union Joint Programming in Neurodegenerative Diseases grant (INTER/JPND/12/01) to Paul Wilmes and an Aide &agrave; la Formation Recherche grant (AFR PHD/4964712) to Cedric C. Laczny all funded by the Luxembourg National Research Fund (FNR).</p>

        <div>
        <img class="large-spacer" src="images/Logo_Uni_quadri_88px.jpg" alt="University of Luxembourg">
        <img class="large-spacer" src="images/fnr.gif" alt="Fonds National de la Recherche">
        <img src="images/LCSB_short_large_RGB_88px.jpg" alt="Luxembourg Centre for Systems Biomedicine">
        </div>


        <footer>
          Vizbin is maintained by <a href="https://github.com/claczny">claczny</a><br>
          This page was generated by <a href="http://pages.github.com">GitHub Pages</a>. Tactile theme by <a href="https://twitter.com/jasonlong">Jason Long</a>.
        </footer>

        
      </div>
    </div>
  </body>
</html>
